gatorgrouper_cli.py

```diff
@@ -61,6 +62,13 @@
             GROUPED_STUDENT_IDENTIFIERS = group_rrobin.group_rrobin_num_group(
                 SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.num_group
             )
+        elif (
+            GG_ARGUMENTS.grouping_method == constants.ALGORITHM_GRAPH
+            and GG_ARGUMENTS.num_group is not constants.DEFAULT_NUMGRP
```

Comment:
This if statement would pass if `GG_ARGUMENTS.num_group` was not specified, and therefore `None`. This is probably not the behavior you want.

Reply:
If `GG_ARGUMENTS.num_group` was not specified, wouldn't it use the default num group?

Reply:
You're right, my bad!